file_store/src/file_info_poller.rs

```diff
@@ -179,7 +187,8 @@ where
     }
 }
 
-impl<Message, State, Parser> ManagedTask for FileInfoPollerServer<Message, State, FileStore, Parser>
+impl<Message, State, Parser> ManagedTask
```

Comment:
I don't remember why I hardcoded `FileStore` in here. But maybe we want to make this generic over `Store` like the impl block below.

That way someone could still use `task-manager` if they decide to implement `FileInfoPollerStore` themselves.